flexWecDesignOpt/main.py

```diff
@@ -53,24 +53,31 @@ def create_case_files(common_bem_file_folder, copy_folder, design_vars):
     def change_case_file(text_file, design_var):
```

Comment:
I think one test idea could involve having a test input file and expected_output file. You could run this function and make sure the observed_output file == expected_output file. Just a thought! Otherwise maybe making sure that the code is correctly location "?" and compare the expected location and the observed location for a test text file? Let me know if that makes sense.

Reply:
Also, I understand what this function is supposed to do from our conversation last week, and I feel like there is a more parsimonious way to accomplish this (maybe not!)? Right now I'm sure the priority it getting a working code together, but if you end up adding a few annotations in the future and are looking for feedback I'd be more than happy to throw around other potential solutions!

Reply:
- Got it! Will definitely implement that test soon, good call
- And I think you're right too, there most definitely is a better way to accomplish this. Will look into it
- Also planning to actually comment out code, that's super on me